tests/container_test.py

```diff
@@ -39,55 +30,9 @@ def test_wait_for_ready(main_container):
             f'Expected "{READY_MESSAGE}" in the log within {TIMEOUT} seconds.'
         )
 
-
-def test_wait_for_exits(main_container, version_container):
-    """Wait for containers to exit."""
-    assert main_container.wait() == 0, "Container service (main) did not exit cleanly"
-    assert (
-        version_container.wait() == 0
-    ), "Container service (version) did not exit cleanly"
```

Comment:
Why are all these tests removed?  Are they not appropriate for this Docker container?

Reply:
Agreed- the output and version checks should remain included here, though the output test would need to be modified.

Reply:
@jsf9k That test was removed as the service is not supposed to exit. It is a web process that is supposed to stay up and running. Hence why I only have the test to make sure it starts up properly.

Reply:
@dav3r so what should I do then? Should I re-add that version container and figure out some way that I can output the version number?

Reply:
I agree that `test_wait_for_exits()` and `test_output()` can be removed since you only have one container and it doesn't exit after startup.  I was confusing `test_output()` with `test_wait_for_ready()` when I wrote my previous comment.

I think `test_release_version()` and `test_container_version_label_matches()` should remain, though you'll have to look at how they work in the skeleton in order to get them working with this repo.

As for `test_log_version()`, is there some kind of version that gets logged in your main container's output that we can verify?

Reply:
So I can modify the dockerfile to add the "org.opencontainers.image.version" tag. Then I can add an echo to etc/entrypoint.sh to output the version as well. Would that work?

Reply:
@dav3r I added tests to cover versioning. However, I had to do it all in a single function as the container never shuts down. Let me know if these latest changes look good to you. https://github.com/cisagov/con-pca-web/pull/145/commits/8c378b04ee7ca1e83daf4f69e8b8b531c4e3b9c5

Reply:
@zenine07 That all makes sense and looks good to me.  The only small things that I would change are:
* `Running Con-PCA Version` --> `Running Con-PCA version` (lowercase v, in both the echo and the test)
* Line 2 of `tests/container_test.py` still mentions the `example` container; that should be updated.

Reply:
Awesome. I have made those small changes.